scripts/build.js

```diff
@@ -28,6 +28,22 @@ schemaFiles.forEach(file => {
 	};
 });
 
+let descriptionBase = "src/description/"
+let descriptionFiles = fs.readdirSync(descriptionBase);
+descriptionFiles.forEach(file => {
+	let raw = fs.readFileSync(descriptionBase + file);
+	const methodName = file.split(".")[0]
+	let stringyDescription = raw.toString();
+	methods.forEach((method, i) => {
+		if (method.name === methodName) {
+			
+			methods[i].description = stringyDescription;
+		}
+	})
+});
```

Comment:
Consider a `for ... of` loop here rather than a `collection.forEach`.  Also recommend using `const` instead of `let` as it provides stronger guarantees to the reader and the compiler about intent.
```suggestion

for (const file of descriptionFiles) {
	const raw = fs.readFileSync(descriptionBase + file);
	const methodName = file.split(".")[0]
	const stringyDescription = raw.toString();
	for (const method of methods) {
		if (method.name === methodName) {
			method.description = stringyDescription;
		}
	}
}
```

Reply:
Though, a `.map` may actually be more appropriate here.